swarm/storage/chunker_test.go

```diff
@@ -234,66 +234,6 @@ func testRandomData(splitter Splitter, n int, tester *chunkerTester) Key {
 	return key
 }
 
-func testRandomDataAppend(splitter Splitter, n, m int, tester *chunkerTester) {
```

Comment:
I have embedded this method as part of the test which uses it, because this method is not used anywhere else. I don't think we need this indirection.